templates/paloalto_panos_test_security-policy-match.textfsm

```diff
@@ -0,0 +1,36 @@
+Value Filldown NAME (.*?)
+Value INDEX (\S+)
+Value Required FROM (\S+)
+Value SOURCE (\S+)
+Value SOURCE_REGION (\S+)
+Value TO (\S+)
+Value DESTINATION ([\S+\s+]+)
+Value DESTINATION_REGION (\S+)
+Value USER (\S+)
+Value SOURCE_DEVICE (\S+)
+Value DESTINATION_DEVICE (\S+)
+Value CATEGORY (\S+)
+Value APPLICATION_SERVICE (\[.*\])
+Value APPLICATION_SERVICE_IMPLICIT (\[.*\])
+Value ACTION (\S+)
+Value ICMP_UNREACHABLE (\S+)
+Value TERMINAL (\S+)
+
+Start
+  ^\"${NAME};\s+index:\s+${INDEX}\"\s+\{
+  ^\s+from\s+${FROM};
+  ^\s+source\s+${SOURCE};
+  ^\s+source-region\s+${SOURCE_REGION}; 
+  ^\s+to\s+${TO};
+  ^\s+destination\s+${DESTINATION};
+  ^\s+destination-region\s+${DESTINATION_REGION};
+  ^\s+user\s+${USER};
+  ^\s+source-device\s+${SOURCE_DEVICE};
+  ^\s+destinataion-device\s+${DESTINATION_DEVICE};
+  ^\s+category\s+${CATEGORY};
+  ^\s+application/service\s+${APPLICATION_SERVICE}];
+  ^\s+application/service\(implicit\)\s+${APPLICATION_SERVICE_IMPLICIT}];
+  ^\s+action\s+${ACTION};
+  ^\s+icmp-unreachable:\s+${ICMP_UNREACHABLE}
+  ^\s+terminal\s+${TERMINAL};
+  ^} -> Record
```

Comment:
Sorry I forgot about this until you said something, but we require the following lines:

```
  ^\s+$$
  ^. -> Error
```

You mentioned more use cases to account for, but you don't need to capture everything. We don't want to put that burden on you, and basically implement what works for you at this point, but you'll have to at least add part of the line to account for it.